frontend/src/app/models/animal.ts

```diff
@@ -0,0 +1,8 @@
+export interface Animal {
+  id: number;
+  title: string;
+  order: string;
+  family: string;
+  createdAt: Date;
```

Comment:
Why do we need `createdAt` and `updatedAt` here? As I remember in sequelize it means `Table creation date` and `Table change date`